sync/src/main/kotlin/gropius/sync/TokenManager.kt

```diff
@@ -145,27 +146,63 @@ abstract class TokenManager<ResponseType : BaseResponseType>(
         return ret
     }
 
+    /**
+     * Check if a user is allowed to be used for syncing
+     *
+     * @param imsProject The IMS to work with
+     * @param user The user to check
+     * @param owner The user that created the data
```

Comment:
I'm a bit confused about the semantics of owner being empty
does this mean that the owner is (for whatever reason) unknown, or does this mean that this is associated with no user?
because this has implications on the `owner.isEmpty()` part below
maybe document these semantics

Reply:
d41ca9e0c667e2be509e81d25615033f61d68351